frontend/src/individual_edit.tsx

```diff
@@ -0,0 +1,424 @@
+      const genebank = genebanks.find(g => g.name == individual.genebank)
+      if (genebank) {
+        return genebank.individuals
+      }
+    }
+    return []
+  }, [genebanks, individual])
+  const motherOptions: OptionType[] = React.useMemo(() => {
+    return genebankIndividuals.filter(i => i.sex == 'female').map(i => {
+      return {value: i.number, label: individualLabel(i)}
+    })
+  }, [genebankIndividuals])
+  const fatherOptions: OptionType[] = React.useMemo(() => {
+    return genebankIndividuals.filter(i => i.sex == 'male').map(i => {
+      return {value: i.number, label: individualLabel(i)}
+    })
+  }, [genebankIndividuals])
+
+  const asIndividual = (number: string | undefined): LimitedIndividual | null => {
+    if (number === null) {
+      return null
+    }
+    const individual = genebankIndividuals.find(i => i.number == number)
+    return individual ? individual : null
+  }
+
+  const removeMeasure = (field: 'weights' | 'bodyfat', index: number) => {
+    console.debug('Deleting ', field, 'number', index)
+    if (individual && individual[field]) {
+      individual[field].splice(index, 1)
+      updateField(field, individual[field])
+    }
+  }
+
+  /**
+   * Fetch individual data from the backend
+   */
+  React.useEffect(() => {
+    user && user.canEdit(id) ?
+      get(`/api/individual/${id}`).then(
+        (data: Individual) => setIndividual(data),
+        error => {
+          console.error(error);
+          userMessage(error, 'error')
+        }
+      )
+      : userMessage('You do not have permission to edit this individual', 'error')
+  }, [id, user])
+
+  /**
+   * Updates a single field in `individual`.
+   *
+   * @param field field name to update
+   * @param value the new value of the field
+   */
+  const updateField = <T extends keyof Individual>(field: T, value: Individual[T]) => {
+    individual && setIndividual({...individual, [field]: value})
+  }
+
+  /**
+   * Sends a request to save the current data in the database. Returns a
+   * ServerMessage.
+   *
+   * @param data The Individual data to save.
+   */
+  const save = (data: Individual) => {
+    const postData = {...data}
+    update('/api/individual', postData).then(
+      (retval: ServerMessage) => {
+        switch (retval.status) {
+          case 'success': userMessage(retval.message ?? 'Individual updated', 'success'); break;
+          default: userMessage(retval.message ?? 'something went wrong', 'error')
+        }
+      },
+      error => {
+        userMessage('' + error, 'error')
+        console.error(error)
+      }
+    )
+  }
+
+  return <>
+  {individual
+    ? <div className={style.form}>
+        <div className={style.flexRowOrColumn}>
+          <div className={style.formPane}>
+            <div className={style.titleText}>
+              Redigera Individ
+            </div>
+            <div className={style.adminPane}>
+              <div className={style.paneTitle}>
+                Admin Fields
```

Comment:
```suggestion
                Kan bara ändras av Genbanksansvarig
```
Can we also have this to expand to the current Genebank Owners name?

Reply:
The second part is low prio and added in #159